triage/update_summaries.sh

```diff
@@ -55,9 +55,11 @@ bq query --allow_large_results --headless -n0 --replace --destination_table k8s-
   where
     test.failed
     and timestamp_to_sec(started) > TIMESTAMP_TO_SEC(DATE_ADD(CURRENT_DATE(), -14, 'DAY'))"
-bq extract --compression GZIP --destination_format NEWLINE_DELIMITED_JSON 'k8s-gubernator:temp.triage' gs://k8s-gubernator/triage_tests.json.gz
-gsutil cp gs://k8s-gubernator/triage_tests.json.gz triage_tests.json.gz
-gzip -df triage_tests.json.gz
+gsutil rm gs://k8s-gubernator/triage_tests/shard_*.json.gz || true
+bq extract --compression GZIP --destination_format NEWLINE_DELIMITED_JSON 'k8s-gubernator:temp.triage' gs://k8s-gubernator/triage_tests/shard_*.json.gz
+mkdir -p triage_tests
```

Comment:
Does the `triage_tests` directory inside the container need to be cleaned up at the end of the script? 

A scenario where I think this would confound results is if the # shards *decreases* between runs. `gzip -drf triage_tests` will *overwrite* shards on subsequent runs but only if they exist.